client/my-sites/checkout/src/components/wp-order-review-line-items.tsx

```diff
@@ -336,6 +339,9 @@ function LineItemWrapper( {
 			return true;
 		}
 
+		if ( shouldRestrict3YearPlans( variant.productSlug ) ) {
```

Comment:
This is fine to do, but you might also want to consider just not returning variants from the shopping cart in this case. The variants returned are all controlled by the shopping cart serializer on the backend. The main reason for this function here is the term length restriction mentioned in the comment at the top.

Reply:
Thanks for the tip!
Since we're now using the billing address to determine the country code, it would make more sense to use the same approach for the variants.

Restricted the changes to the Plan Grid and moved the variants filtering to the backend in D149866-code.